src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -133,13 +133,16 @@ function paginatePages(page, files, field, layout, ariaLabel, perPage) {
 }
 
 // Return page object with path, breadcrumb and title set.
-function updateEntityUrlObj(page, drupalPagePath, title) {
-  const pathSuffix = title.replace(/\s+/g, '-').toLowerCase();
+function updateEntityUrlObj(page, drupalPagePath, title, pathSuffix) {
+  pathSuffix = pathSuffix || title.replace(/\s+/g, '-').toLowerCase();
   let generatedPage = Object.assign({}, page);
-  generatedPage.entityUrl.breadcrumb.push({
-    url: { path: drupalPagePath },
-    text: page.title,
-  });
+  generatedPage.entityUrl.breadcrumb = [
```

Comment:
i want to make sure we're not mutating the original page object by creating a brand new array.